For jina clip v2 release tmrw (#1309)
qdrant/landing_page: qdrant-landing/content/documentation/embeddings/jina-embeddings.md

+ **text-matching**: Used to encode text for similarity matching, such as measuring similarity between two sentences.
+ **separation**: Used for clustering or reranking tasks.

> `jina-embeddings-v3` and `jina-clip-v2` supports **Matryoshka Representation Learning**, allowing users to control the embedding dimension with minimal performance loss.  

Reviewer: ```suggestion
`jina-embeddings-v3` and `jina-clip-v2` support **Matryoshka Representation Learning**, allowing users to control the embedding dimension with minimal performance loss.  
```

)

```

### Jina CLIP v2

> The code below demonstrate how to use `jina-clip-v2` together with Qdrant:

Reviewer: ```suggestion
The code below demonstrates how to use `jina-clip-v2` with Qdrant:
```

```python
import requests
from qdrant_client import QdrantClient
from qdrant_client.models import Distance, VectorParams, PointStruct

# Provide your Jina API key and choose the model.
JINA_API_KEY = "jina_xxxxxxxxxxx"
MODEL = "jina-clip-v2"
DIMENSIONS = 1024  # Set the desired output vector dimensionality.

# Define the inputs
text_input = "A blue cat"
image_url = "https://i.pinimg.com/600x315/21/48/7e/21487e8e0970dd366dafaed6ab25d8d8.jpg"

# Get embeddings from the Jina API
url = "https://api.jina.ai/v1/embeddings"
headers = {
    "Content-Type": "application/json",
    "Authorization": f"Bearer {JINA_API_KEY}",
}
data = {
    "input": [
        {"text": text_input},
        {"image": image_url},
    ],
    "model": MODEL,
    "dimensions": DIMENSIONS,
}

response = requests.post(url, headers=headers, json=data)
response_data = response.json()["data"]

> # Extract embeddings
> text_embedding = response_data[0]["embedding"] #jina-clip-v2 model doesn't differentiate between images and text, so we just assign them based on the order
> image_embedding = response_data[1]["embedding"]

Reviewer: ```suggestion
# The model doesn't differentiate between images and text, so we extract output based on the input order.
text_embedding = response_data[0]["embedding"]
image_embedding = response_data[1]["embedding"]
```

> # Initialize Qdrant client
> client = QdrantClient(":memory:")  # Use ":memory:" for in-memory storage or specify a URL for a persistent instance.

Reviewer: ```suggestion
# Initialize Qdrant client
client = QdrantClient(url="http://localhost:6333/")
```

Reviewer: We should prefer Qdrant server in examples.

# Create a collection with named vectors
collection_name = "MyCollection"
client.recreate_collection(
    collection_name=collection_name,
    vectors_config={
        "text_vector": VectorParams(size=DIMENSIONS, distance=Distance.DOT),
        "image_vector": VectorParams(size=DIMENSIONS, distance=Distance.DOT),
    },
)

> # Upsert the point with named vectors
> point_id = 0  # Unique identifier for the point, UUID is also a good choice
> 
> client.upsert(
>     collection_name=collection_name,
>     points=[
>         PointStruct(
>             id=point_id,
>             vector={
>                 "text_vector": text_embedding,
>                 "image_vector": image_embedding,
>             }
>         )
>     ],
> )

Reviewer: ```suggestion
client.upsert(
    collection_name=collection_name,
    points=[
        PointStruct(
            id=0,
            vector={
                "text_vector": text_embedding,
                "image_vector": image_embedding,
            }
        )
    ],
)
```

# Now let's query the collection
search_query = "A purple cat"

# Get the embedding for the search query from the Jina API
url = "https://api.jina.ai/v1/embeddings"
headers = {
    "Content-Type": "application/json",
    "Authorization": f"Bearer {JINA_API_KEY}",
}
data = {
    "input": [{"text": search_query}],
    "model": MODEL,
    "dimensions": DIMENSIONS,
    # "task": "retrieval.query" # Uncomment this line for text-to-text retrieval tasks
}

response = requests.post(url, headers=headers, json=data)
query_embedding = response.json()["data"][0]["embedding"]

> search_results = client.search(
>     collection_name=collection_name,
>     query_vector=("image_vector", query_embedding),  # Search against image vectors
>     limit=5
> )

Reviewer: ```suggestion
search_results = client.query_points(
    collection_name=collection_name,
    query=query_embedding,
    using="image_vector",
    limit=5
).points
```

Reviewer: We recommend the query API now.
